addons/sale/models/sale_order_line.py

```diff
@@ -261,7 +261,7 @@ def write(self, values):
 
     discount = fields.Float(
         string='Discount (%)', digits='Discount',
-        compute='_compute_discount', store=True, readonly=False, precompute=True)
+        compute='_compute_discount', store=True, readonly=False)
```

Comment:
We have a big perf loss if price_unit is precomputed and discount is post-computed, since they both rely on the pricelist logic and the magic `pricelist_item_id` field used as pricelist 'cache'.
Shouldn't we put the price_unit post-compute as well?  Because it also has a digits field and probably suffers from the same problem :thinking:

Reply:
we can do that but then we have to remove the required on the field

Reply:
Gosh, forgot this point :man_facepalming:.